homeassistant/components/binary_sensor/icons.json

```diff
@@ -0,0 +1,207 @@
+{
+  "entity_component": {
+    "_": {
+      "icon": "mdi:radiobox-blank",
+      "state": {
+        "off": "mdi:radiobox-blank",
```

Comment:
We should remove all duplicate lines if possible.
In this concrete example we don't need to specify the `off` state as it will fallback to the icon specified in the `icon` key, which is the same

```suggestion
```